shared-bindings/wifi/Radio.c

```diff
@@ -580,11 +640,25 @@ static MP_DEFINE_CONST_FUN_OBJ_KW(wifi_radio_set_ipv4_address_obj, 1, wifi_radio
 //|         """Sets the IP v4 address of the access point. Must include the netmask and gateway."""
 //|         ...
 static mp_obj_t wifi_radio_set_ipv4_address_ap(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
-    enum { ARG_ipv4, ARG_netmask, ARG_gateway };
+    enum
+    {
+        ARG_ipv4,
+        ARG_netmask,
+        ARG_gateway
+    };
     static const mp_arg_t allowed_args[] = {
-        { MP_QSTR_ipv4, MP_ARG_REQUIRED | MP_ARG_KW_ONLY | MP_ARG_OBJ, },
-        { MP_QSTR_netmask, MP_ARG_REQUIRED | MP_ARG_KW_ONLY | MP_ARG_OBJ, },
-        { MP_QSTR_gateway, MP_ARG_REQUIRED | MP_ARG_KW_ONLY | MP_ARG_OBJ, },
+        {
+            MP_QSTR_ipv4,
+            MP_ARG_REQUIRED | MP_ARG_KW_ONLY | MP_ARG_OBJ,
+        },
+        {
+            MP_QSTR_netmask,
+            MP_ARG_REQUIRED | MP_ARG_KW_ONLY | MP_ARG_OBJ,
+        },
+        {
+            MP_QSTR_gateway,
+            MP_ARG_REQUIRED | MP_ARG_KW_ONLY | MP_ARG_OBJ,
+        },
     };
```

Comment:
Does uncrustify still change all of this?

Reply:
Still working on this.  The micropython-uncrustify package from requirements_dev changes it, and so does manually calling  `uncrustify -c tools/uncrustify.cfg  --replace shared-bindings/wifi/Radio.c` with the old archived Ubuntu 22.04 binary(Which is 0.72).  Modern versions just give a ton of warnings about the deprecated options and do the same thing.

Reply:
I used Meld to pick the changes one by one and clean up the stuff the bad IDE settings messed up, micropython-uncrustify seems to be working now